package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hubot-cf-convenience",
-  "version": "0.0.7",
+  "version": "0.0.8",
```

Comment:
Should we always be bumping the version in package.json, or might that be getting done at some other automated level, like when we decide to push to npm?